generator/osm2meta.cpp

```diff
@@ -289,6 +289,13 @@ string MetadataTagProcessorImpl::ValidateAndFormat_wikipedia(string v) const
 
 string MetadataTagProcessorImpl::ValidateAndFormat_wikimedia_commons(string v) const
 {
+
+    // Putting the full wikimedia url to this tag is incorrect according to:
+    // https://wiki.openstreetmap.org/wiki/Key:wikimedia_commons
+    // But it happens often enough that we should guard against it.
+    strings::ReplaceFirst(v, "https://commons.wikimedia.org/wiki/", "");
```

Comment:
Actually, using strings::StartsWith + std::string::erase will be faster than this implementation.